ocs_ci/ocs/cluster.py

```diff
@@ -1551,6 +1552,35 @@ def is_flexible_scaling_enabled():
     return failure_domain == "host" and flexible_scaling
 
 
+def validate_osd_pdb():
```

Comment:
the function names as validate and we are returning list, should we return bool instead?